### ISSUE TITLE
Retrieve the Ltree data naturally

### ISSUE THREAD

**Issue by kivikakk** [kivikakk/diesel_ltree#1]
Currently blocked by [`libpq` of all things](https://www.postgresql.org/docs/9.1/static/libpq-exec.html):

> <dl><dt><code>resultFormat</code></dt><dd>Specify zero to obtain results in text format, or one to obtain results in binary format. (There is not currently a provision to obtain different result columns in different formats, although that is possible in the underlying protocol.)</dd></dl>

Ideally we'd request text format just for `ltree`, since requesting binary produces the following from Postgres:

``` rust
DatabaseError(__Unknown, "no binary output function available for type ltree")
```

**Comment by kivikakk** [comment]
wontfix

**Comment by gtors** [comment]
Why wontfix?

**Comment by kivikakk** [comment]
I last worked on this (in earnest) nearly two years ago, and this issue is blocked by `libpq`. If you want to fix `libpq` and then create a PR here, by all means, please do!
